### PR TITLE
OSDOCS#8830:Updated OCM and HCC attributes and references to them in ROSA docs

### PR DESCRIPTION
**Pull request by AedinC**
Version(s):
4.14+

Issue:
[OSDOCS-8830](https://issues.redhat.com/browse/OSDOCS-8830)

Link to docs preview:

- [Red Hat OpenShift Service on AWS quick start guide](https://71722--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-quickstart-guide-ui)
- [Creating a ROSA cluster with AWS STS using the default auto mode](https://71722--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-quickstart-guide-ui#rosa-quickstart-creating-a-cluster)
- [Deploying an application from the Developer Catalog](https://71722--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-quickstart-guide-ui#deploy-app_rosa-quickstart-guide-ui)
- [Overview of the default cluster specifications](https://71722--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly#rosa-sts-overview-of-the-default-cluster-specifications_rosa-hcp-sts-creating-a-cluster-quickly)

QE review:
- [x] QE has approved this change.

Peer review:
- [x] Peer reviewer has approved this change.

SME review:
- [x] SME has approved this change.

### CODE REVIEW

**Comment by AedinC** [comment]
@abhgupta Can you please review this PR? It addresses some inconsistencies in how we refer to OCM in the HCC in ROSA docs-many thanks!

File: rosa_getting_started/rosa-quickstart-guide-ui.adoc [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+= {product-title} quick start guide
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-quickstart-guide-ui
 
 toc::[]
 
 [NOTE]
 ====
-If you are looking for a comprehensive getting started guide for ROSA, see xref:../rosa_getting_started/rosa-getting-started.adoc#rosa-getting-started[Comprehensive guide to getting started with {product-title}]. For additional information on ROSA installation, see link:https://www.redhat.com/en/products/interactive-walkthrough/install-rosa[Installing Red Hat OpenShift Service on AWS (ROSA) interactive walkthrough].
+If you are looking for a comprehensive getting started guide for {product-rosa} (ROSA), see xref:../rosa_getting_started/rosa-getting-started.adoc#rosa-getting-started[Comprehensive guide to getting started with {product-title}]. For additional information on ROSA installation, see link:https://www.redhat.com/en/products/interactive-walkthrough/install-rosa[Installing Red Hat OpenShift Service on AWS (ROSA) interactive walkthrough].
```
Comment (kcarmichael08) [inline rosa_getting_started/rosa-quickstart-guide-ui.adoc:11]: I think `{product-rosa}` is missing from the common attributes. Should it be `{product-title}`?
Screenshot 2024-02-16 at 2 59 34 PM

**Review by kcarmichael08 — COMMENTED** [review]
One issue/question; otherwise, LGTM!

File: rosa_getting_started/rosa-quickstart-guide-ui.adoc (hunk 1 above)
Comment (AedinC) [reply rosa_getting_started/rosa-quickstart-guide-ui.adoc:11]: Good catch-thanks @kcarmichael08!

**Comment by AedinC** [comment]
Hi @dtaylor113, would you or someone on your team be able to take a look at this to ensure we (the ROSA docs team) are referring correctly to OCM in the UI? Many thanks.

File: modules/deploy-app.adoc [hunk 2]
```diff
@@ -20,15 +20,15 @@ From the {product-title} web console, you can deploy a test application from the
...
+* You logged in to the {hybrid-console-url}.
+* You created a {product-title} cluster.
 * You configured an identity provider for your cluster.
 * You added your user account to the configured identity provider.
 endif::[]
 
 .Procedure
 
-. From the {cluster-manager} {hybrid-console-second}, click *Open console*.
+. From the *Clusters* page of the {hybrid-console-second}, click *Open console*.
```
Comment (jmekkatt) [inline modules/deploy-app.adoc:31]: In Clusters page there is no option called "Open console". "Open console" will be  shown as context menu of a selected cluster. Refer inline screenshot
image
I would request to update the step definition as per above mentioned flow.

File: modules/deploy-app.adoc [hunk 3]
```diff
@@ -20,15 +20,15 @@ From the {product-title} web console, you can deploy a test application from the
...
 
 . In the *Administrator* perspective, select *Home* -> *Projects* -> *Create Project*.
```
Comment (jmekkatt) [inline modules/deploy-app.adoc:33]: Before this step, User will be redirected to openshift cluster console page and need to login with configured IDP credentials. Probably worth to mention the same step? Please review it.

File: modules/deploy-app.adoc (hunk 2 above)
Comment (AedinC) [reply modules/deploy-app.adoc:31]: Thanks @jmekkatt, I update that.

**Comment by AedinC** [comment]
@jmekkatt Can you please re-review updated content and approve if no more changes are to be made? Thanks.

**Review by ciaranRoche — COMMENTED** [review]
lgtm

**Comment by AedinC** [comment]
> lgtm

Thanks @ciaranRoche

**Comment by jmekkatt** [comment]
@AedinC Thanks for the change.
/lgtm

**Pull request merged**